src/aics-image-viewer/components/App/index.tsx

```diff
@@ -304,9 +329,35 @@ const App: React.FC<AppProps> = (props) => {
         // now manually remap control points using the channel's old range
         const { controlPoints } = thisChannelsSettings;
         const newControlPoints = remapControlPointsForChannel(controlPoints, oldRange, thisChannel);
+        currentControlPoints = newControlPoints;
         changeChannelSetting(channelIndex, "controlPoints", newControlPoints);
       }
     }
+
+    // Save the first loaded channel's volume subregion as our reference for resetting channel states.
+    if (!getSavedSubregionSize()) {
+      setSavedSubregionSize(aimg.imageInfo.subregionSize.clone());
+    }
```

Comment:
This and other API changes are explained in `ViewerStateProvider/types.ts` and `ViewerStateProvider/ResetSTateProvider.tsx`.